.gitignore

```diff
@@ -24,3 +24,7 @@ _testmain.go
 *.prof
 
 tmp
+
+.idea/
+.vscode/
+.DS_Store
```

Comment:
I have a strong opinion about such changes

- https://github.com/ccoVeille/go-safecast/pull/35#pullrequestreview-2367333706

TL; DR; this varies too much between people, use git config `core.excludeFiles` setting locally.